static/js/global.js

```diff
@@ -0,0 +1,24 @@
+var browser_locale = window.navigator.language.split('-')[0]
+
+function updateTranslation(localex) {
+  $.i18n({
+    locale: localex,
+  })
+
+  $.i18n()
+    .load('./js/i18n', localex)
+    .done(function() {
+      console.log('i18n:' + localex + ' locale loaded')
+      $('body').i18n()
+      $('html').attr('lang', localex)
+      $.getScript('js/app.js')
+    })
+}
+
+$(window).on('load', function() {
+  updateTranslation(browser_locale)
+  $('#lang-select').val(browser_locale)
+  $('#lang-select').on('change', function() {
+    updateTranslation(this.value)
```

Comment:
It would be nice if the user's preference is saved (preferably in localStorage), so that the user doesn't need to change it on every visit.

Reply:
Also why is this not in the other file? (language_switcher)